packaging/Microsoft.NET.Sdk.WindowsDesktop/targets/Microsoft.NET.Sdk.WindowsDesktop.targets

```diff
@@ -1,20 +1,8 @@
 <Project>
-
-   <!-- 
-    For now, UseLegacyPresentationBuildTasks=true can be used to use the in-box framework 
-    PresentationBuildTasks and targets when running in full framework MSBuild. This is
-    just an escape hatch while the new PresentationBuildTasks bake. 
-    
-    When we remove this,we can remove the correpsonding corresponding mscorlib swap targets in
-    Microsoft.DesktopUI.App.targets, and the production of ref-mscorlib shims 
-  -->
- <Import Project="$(MSBuildToolsPath)\Microsoft.WinFX.targets" 
-         Condition ="'$(MSBuildRuntimeType)' != 'Core' and '$(UseLegacyPresentationBuildTasks)' == 'true'"/>
-
- <Import Project="Microsoft.WinFX.targets" 
-         Condition="'$(MSBuildRuntimeType)' == 'Core' or '$(UseLegacyPresentationBuildTasks)' != 'true'" />
-
-  <ItemGroup Condition=" '$(EnableDefaultItems)' == 'true' And '$(UseWPF)' == 'true'">
+  <Import Project="Microsoft.WinFX.targets" />
+  
+  <!-- $(TargetFrameworkIdentifier) can be '.NETCoreApp' or '.NETFramework' -->
```

Comment:
Elaborate. 

Explain that WPF is supported on WindowsDesktop SDK on .NET Core 3.0+, or .NET Framework 3.0+.